exporter/exporter.go

```diff
@@ -169,9 +169,9 @@ func Run(args ...string) error {
 	}
 
 	// default to public API environment
-	lightstepEnv := "public"
-	if len(os.Getenv("LIGHTSTEP_ENV")) > 0 {
-		lightstepEnv = os.Getenv("LIGHTSTEP_ENV")
+	lightstepUrl := "https://api.lightstep.com"
```

Comment:
🎨 Hoist this up as a constant

Reply:
I brought this logic in from the client.go file and left a comment to remove it once `environment` is deprecated. I'd rather leave the URL in-line to make it easier to clean up later. WDYT?

Reply:
Yeah, the only concern would be that this URL is mentioned in multiple places and things diverge. If the risk is low there, feel free to ignore.